ansible/group_vars/artifactory/vars

```diff
@@ -1,2 +1,2 @@
 ---
-artifactory_version: 7.63.12
+artifactory_version: 7.71.4
```

Comment:
This is redundant as `artifactory_version` is already defined and passed in to Ansible via the Packer config